src/Serilog.Sinks.Email/Serilog.Sinks.Email.csproj

```diff
@@ -5,6 +5,9 @@
     <Authors>Serilog Contributors</Authors>
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net462;net471</TargetFrameworks>
```

Comment:
is the net471 for a reason ?

Reply:
None at all; sorting it out now, thanks :-)